src/Filters/atoms/DateTimeBlock.tsx

```diff
@@ -126,6 +145,13 @@ const DateTimeBlock : React.FC<IProps> = ({
   }, [date, displayMinutes, setDateCallback]);
 
   const setDateMinutes = useCallback(({target: {value}}: React.ChangeEvent<HTMLInputElement>) => {
+    const isContainsSymbolN = /^[0-5]{0,1}[0-9]$/;
+    if(Number(value)>59){
+      value = '59';
+    } else if (!isContainsSymbolN.test(value) && value !== '') {
+      return;
+    }
```

Comment:
Please reconsider this checks, I think you only need RegEx check.

Reply:
In regex we can't do this thing like..
if value is more that 59 like 88
it must add only 59 so I have added another if statement.
Please tell me your thoughts.

Reply:
I think regex is capable of excluding number greater than 59.